R/var_labels.R

```diff
@@ -0,0 +1,15 @@
+#' Creates a `tbl_df` that contains the column names and variable labels to
+#' easily explore the data.
+#' @export
+#' @title var_labels
+#' @param data a data.frame
+var_labels <- function(data) {
+  if("data.frame" %in% class(data) == FALSE){
```

Comment:
`!is.data.frame()` is a little simpler. It's smart enough to check for any parent classes.